ecoli/analysis/single/fba_flux_pca.py

```diff
@@ -0,0 +1,380 @@
+"""
+Visualize FBA reaction flux dynamics using PCA trajectory analysis.
```

Comment:
Interesting idea. Do you have any examples of reaction IDs that yielded interesting plots?